draft-ietf-uuidrev-rfc4122bis.md

```diff
@@ -1189,7 +1189,7 @@ Truncating:
 
 Error Handling:
 : If a system overruns the generator by requesting too many UUIDs
-  within a single system time interval, the UUID service MUST either
+  within a single system time interval, the UUID service SHOULD either
```

Comment:
My concern on this one is that stalling the clock or erroring are not necessarily the only viable solutions to this problem.  But I'm on the fence on this one.  I think I also added separately in another PR a "MUST NOT" sentence that says the thing that it really cannot do is return a knowingly duplicated value, which I think is the key point to stress here.